.github/workflows/ci.yml

```diff
@@ -21,11 +21,13 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         include:
-          - python-version: "3.8"
-          - python-version: "3.9"
           - python-version: "3.10"
+            djangomain: true
           - python-version: "3.11"
+            djangomain: true
```

Comment:
Mmm, I think it would be neat to have 5 or 6 entries in this list:

* 3.8 "vanilla"
* 3.9 "vanilla"
* 3.10 "vanilla"
* 3.11 "vanilla"
* (3.10 djangomain) (not sure this is a priority)
* 3.11 djangomain

Then the djangomain steps wouldn't need to be set as `continue-as-error` and we'd immediately see a failing/succeeding djangomain check in the PR's checks list without having to dig into the check outputs?

Reply:
Ah good point! I would say that 3.10 is a priority since it is the minimum version for django main right now and what django 5.0 is going to support.

Reply:
I think differences in 3.10 and 3.11 support should be adequately covered by the vanilla 3.10/3.11 tests, but I'm fine with 6 tests instead of 5!

Reply:
Ah yes, true. But yeah, let's keep them so we are covered. You never know :stuck_out_tongue_closed_eyes: